### ISSUE TITLE
Blank spell: /ma " III" <t>

### ISSUE THREAD

**Issue by momotaroko** [ekrividus/autoMB#4]
If I set tier 5 for Double Burt, it would work and do this:

```
/ma "Thunder V" <t>
/ma "Thunder IV" <t>
```

Then a new skillchain is appearing while Thunder V and IV is cooling down it will try to burst lower tier Thunder III as it should, but it does an error 95% of the time and shows a blank spell with this message so no burst is occurring:

`/ma " III" <t>`

**Comment by ekrividus** [comment]
Added to the list of things to look into when I rejoin the game in spring.

**Comment by ekrividus** [comment]
Haven't been able to recreate this. If you do please share how.

**Comment by momotaroko** [comment]
I was able to recreate it by having these settings:

Settings:
- tier 6
- doubleburst on
- stepdown always

Events:
1. Light skillchain appears.
2. Double burst occurs correctly, casting Thunder VI followed by Thunder V.
3. Spells timers of Thunder VI and Thunder V is cooling down.
4. Another Light skillchain appears.
5. It will have error "/ma "<blank> IV <t>"

---
Settings:
- tier 5
- doubleburst on
- stepdown always

Events:
1. Light skillchain appears.
2. Double burst occurs correctly, casting Thunder V and Thunder IV.
3. Spells timers of Thunder V and Thunder IV is cooling down.
4. Another Light skillchain appears.
5. It will have error "/ma "<blank> III <t>"

---

Settings:
- tier 4
- doubleburst on
- stepdown always

Events:
It will never have error "/ma "<blank> II <t>", because the spells Thunder IV and Thunder III cool down fast therefore are "always" ready.

---

To work around it, I have set "stepdown never", so it will swap to another spell to double burst.

**Comment by ekrividus** [comment]
What areas was this happening in? Solo, party, full alliance? 
It tried to make this happen spamming SCH scs and during a Dyna run trying with T6/5/4/3 and never had this happen. With multiple SCs popping back to back it would go all the way down to T2 spells without missing any. 
I'm wondering if lag was a factor, or if I accidently fixed this when sorting out another issue.
If you try the most recently pushed version and it's still happening do 
//amb debug
//amb toggle spell
//amb toggle weather

It'll spam your logs a bit but hopefully it'll also narrow down exactly where it's failing to choose a spell.
And tell me if there's anything really odd in there.

**Comment by ekrividus** [comment]
I finally got this to replicate. Though I was on Nin when it happened. 
Sadly it was so intermittent that I couldn't put together a specific set of events to make it happen regularly. 
I'm gonna keep watching to see if I can pick out a pattern though.

**Comment by rukuwa00** [comment]
Have this problem when I am DDing on Pld first light SC it double MBs thunder 6 and 5 np but when the second light SC, it have the same /ma  "IV" <t> error

**Comment by ekrividus** [comment]
Yeah I was never able to get this to occur in a way that helped me sort out what was causing it. Sadly I don't have any active accounts at the moment so I can't do much to dig into a fix. When I come back though it is for sure on my list of things to sort out.
